.gitignore

```diff
@@ -0,0 +1,14 @@
+/debian/*.debhelper.log
```

Comment:
Hilariously, you have been shamed by the reuse gods you are trying to appease:

```
>>> MALLOC_PERTURB_=186 /usr/bin/reuse lint
――――――――――――――――――――――――――――――――――――― ✀  ―――――――――――――――――――――――――――――――――――――
# MISSING COPYRIGHT AND LICENSING INFORMATION

The following files have no copyright and licensing information:
* .gitignore
```

That seems pretty pedantic.

Reply:
It's beautiful.

I regret trying to make this work.

Reply:
The options are:

1. Add a license comment to this obviously non-creative work.
2. Add the same information to `.reuse/dep5`

I took option 1.